src/components/document/document-annotation-toolbar.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { observer } from "mobx-react-lite";
+import classNames from "classnames";
```

Comment:
A nit, but it can be useful to alphabetize imports:
```suggestion
import classNames from "classnames";
import { observer } from "mobx-react-lite";
import React from "react";
```